testing/file_test/README.md

```diff
@@ -217,6 +217,8 @@ Supported comment markers are:
 
     -   `// ARGS`
     -   `// EXTRA-ARGS`
+        -   Included `EXTRA-ARGS` will come before `EXTRA-ARGS` in the main
```

Comment:
```suggestion
        -   Included `EXTRA-ARGS` will come before `EXTRA-ARGS` in the primary test
```
perhaps? `main` is a bit overloaded

Reply:
As noted, I think the code is already referring to this as the "main file" for tests. I'd like to stick with the same phrasing here for consistency.